### PR TITLE
OWNERS, kubevirt: remove cluster-up OWNERS file

### PR DESCRIPTION
**Pull request by dhiller**
### What this PR does
Before this PR:

cluster-up folder contains an OWNERS file which needs to be restored on every bump.

After this PR:

Removed OWNERS file from cluster-up folder, since now root OWNERS file defines ownership. Also removed the changes from [previous PR for sig-buildsystem](https://github.com/kubevirt/kubevirt/pull/11183) that restore the OWNERS since it's no more necessary.

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Special notes for your reviewer

/cc @fabiand @xpivarc 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: OWNERS
Comment (dhiller) [inline OWNERS:12]: @xpivarc did you mean like so?

Comment (xpivarc) [reply OWNERS:12]: Yes

**Review by xpivarc — COMMENTED** [review]
/approve

**Comment by dhiller** [comment]
Can I get an `lgtm` here before I have to rebase again, please?

**Comment by dhiller** [comment]
@xpivarc @acardace  could I get an `lgtm` before I need to rebase once again, please? Thank you!

**Review by xpivarc — APPROVED** [review]

**Comment by dhiller** [comment]
/override pull-kubevirt-e2e-kind-1.27-vgpu

Irrelevant for this change.

**Pull request merged**
